src/style.css

```diff
@@ -0,0 +1,19 @@
+* {
+  margin: 0;
+}
```

Comment:
don't overuse this selector, this is mentioned in the checklist and I wrote you before, this selector mostly used to define box-sizing property